### DISCUSSION TITLE
Where are the preview/experimental releases?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WindowsAppSDK#2204] [Q&A]
The [current cadence documentation](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/release-channels) states that Preview and Experimental channels target monthly releases. The Windows App SDK team has now missed **5 releases**, give or take a few for "targeting" fudge and/or the holidays.

Can you please acknowledge the delay and provide an update on the state of the Windows App SDK? Is there a new schedule you can share?

**Comment by dongle-the-gadget** [comment]
Yeah I'm so PUMPED to get my hands on the new WASDK 1.1 (even in an early stage) for Mica and multi window support.

**Reply by yoshiask** [reply]
I'm pretty sure mica/host backdrop acrylic aren't planned for 1.1 anymore. The productboard site only has "background acrylic", and it's "under consideration".

**Reply by riverar** [reply]
They just updated the roadmap (in `docs\`) to say it's coming. So not clear which one is right. I opened an issue in hopes of getting clarification. (#2271)

**Comment by ghost1372** [comment]
Hi @gabbybilka 
Can you tell us when WASDK 1.1-preview will be released?
We have been waiting for several months

**Reply by riverar** [reply]
1.0.1 was released so a preview doesn't make sense. Maybe you meant 1.1-preview?

**Reply by ghost1372** [reply]
> 1.0.1 was released so a preview doesn't make sense. Maybe you meant 1.1-preview?

Yes, typo issue😅

**Reply by dimohy** [reply]
There is a strong need for elevation of privilege. When can we expect a fast 1.0.2 or 1.1-preview?
